File: lista python/at1.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def caracteristicas_carro(self):
        return f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}"
    

carro0 = Carro("Fiat", "Mobi", 2024)
carro1 = Carro("Volkswagen", "Gol", 2024)
carro2 = Carro("Hyundai", "HB20", 2024)

print(carro0.caracteristicas_carro())
print(carro1.caracteristicas_carro())
print(carro2.caracteristicas_carro())

File: lista java/at1.java
public class Carro {
    
    private String marca;
    private String modelo;
    private int ano;

    
    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    
    public void exibirDetalhes() {
        System.out.println("Marca: " + marca + ", Modelo: " + modelo + ", Ano: " + ano);
    }

    public static void main(String[] args) {
        
        Carro carro1 = new Carro("Fiat", "Mobi", 2024);
        Carro carro2 = new Carro("Volkswagen", "Gol", 2024);
        Carro carro3 = new Carro("Hyundai", "HB20", 2024);

        
        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}
